packages/ui/v2/core/Shell.tsx

```diff
@@ -143,12 +144,26 @@ const Layout = (props: LayoutProps) => {
 
       {/* todo: only run this if timezone is different */}
       <TimezoneChangeDialog />
-
-      <div className="flex h-screen overflow-hidden" data-testid="dashboard-shell">
-        {props.SidebarContainer || <SideBarContainer />}
-        <div className="flex w-0 flex-1 flex-col overflow-hidden">
-          <ImpersonatingBanner />
-          <MainContainer {...props} />
+      <div className="h-screen overflow-hidden">
+        <TopBanner
+          actions={
+            <button
+              className="border-b border-black text-sm"
+              onClick={() => {
+                console.log("test");
+              }}>
+              Action
+            </button>
+          }
+          variant="warning"
+          text="An app wide warning is occuring that should be resolved."
+        />
```

Comment:
On which conditions will this component be rendered/hidden?

Reply:
discussion about this here https://threads.com/34433601299?s=v45qV4JNriP5dLaqWvNqNm. this is just for testing how the component would look like